Shared/Samples/Show realistic light and shadows/README.md

```diff
@@ -0,0 +1,30 @@
+# Show realistic light and shadows
+
+Show realistic lighting and shadows for a given time of day.
+
+![Image of realistic lighting and shadows](realistic-lighting-and-shadows.png)
```

Comment:
```suggestion
![Image of Show realistic light and shadows sample](show-realistic-light-and-shadows.png)
```
Please fix the name. (one of the purposes of having "URL to README" in the PR template 😏